README.md

```diff
@@ -1,20 +1,17 @@
 # WiBo
 
-An experiment to try and write a minimal, low-fuss wrapper that can run really simple command-line 32-bit Windows binaries on Linux - with less faff and less dependencies than WINE.
+A minimal, low-fuss wrapper that can run really simple command-line 32-bit Windows binaries on Linux - with less faff and less dependencies than WINE.
 
 Don't run this on any untrusted executables, I implore you. (Or probably just don't run it at all... :p)
 
     make && ./wibo
 
-If you need something like this project (but more mature), you might find [taviso/loadlibrary](https://github.com/taviso/loadlibrary) more interesting.
```

Comment:
I found this reference useful/interesting fwiw

Reply:
I'll add a related projects section at the bottom